src/panels/lovelace/editor/unused-entities/hui-unused-entities.ts

```diff
@@ -42,7 +42,8 @@ export class HuiUnusedEntities extends LitElement {
   @property() public hass?: HomeAssistant;
 
   private _unusedEntities: string[] = [];
-  @property() private _filteredUnusedEntities: string[] = [];
+  private _filteredUnusedEntities: string[] = [];
+  @property() private _sortedUnusedEntities: string[] = [];
```

Comment:
Instead of keeping track of it in an instance variable, use memoizeOne. It will create a sort function that always remembers the last output. Just checks if input variables are the same and not do any work if it is. So it will only filter it once for every set of entities. 

Example: https://github.com/home-assistant/home-assistant-polymer/blob/dev/hassio/src/addon-store/hassio-addon-repository.ts#L28-L37